cli-feed-v4.json

```diff
@@ -10442,6 +10442,121 @@
+              "version": "1.0.3"
+            },
+            "default": false,
+            "toolingSuffix": "net5-isolated",
+            "localEntryPoint": "dotnet.exe",
+            "targetFramework": "net5.0",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ItemTemplates/3.1.2185",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.Functions.Worker.ProjectTemplates/3.1.2185",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.3.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet-isolated:3.0-dotnet-isolated5.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~3",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v4.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET-ISOLATED|5.0"
+            }
+          }
+        }
+      },
+      "coreTools": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.4837/Azure.Functions.Cli.linux-x64.3.0.4837.zip",
+          "sha2": "8fa72a13fc07655d650886f5c3fda9d72815d1d2a994ccbea9ced11f10f4a642",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.4837/Azure.Functions.Cli.osx-x64.3.0.4837.zip",
+          "sha2": "c5e537d6091789a91db4c379abf814a596f296f71b9c6610dd0e44635d8825c8",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.4837/Azure.Functions.Cli.min.win-x64.3.0.4837.zip",
+          "sha2": "19d397eb5a2c64d8101c7dbb0d55b5e8eed0dfa9ec7e53a5f47e7d6cdad89dd3",
+          "size": "minified",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.4837/Azure.Functions.Cli.win-x86.3.0.4837.zip",
+          "sha2": "90ea6d45db4b10e5e296fd39073660cb405c782723b290a274c72fab374c9587",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/3.0.4837/Azure.Functions.Cli.min.win-x86.3.0.4837.zip",
+          "sha2": "41fa6476c42c8171d27799f5f14ba209f41360eb8e0218c7ae2a1bb3b7251fb0",
+          "size": "minified",
+          "default": "true"
+        }
+      ]
     }
   }
```

Comment:
I think you're missing a closing brace, making this invalid json. Can you double check please